cmd/metricsbridge/metricsbridge.go

```diff
@@ -233,6 +241,15 @@ func (c *config) runOnce(ctx context.Context) error {
 			for k, v := range sample.Metric {
 				if k != model.MetricNameLabel {
 					f.Dims[string(k)] = string(v)
+					//if there's an instance dimension try to lookup the hostname
+					if string(k) == "instance" {
```

Comment:
nit: to my eye it'd be neater and clearer to move this out of the for loop and do `if instance, found := sample.Metric["instance"]; found {`

Reply:
moved below, among other additional dimensions